blogs/cli/index.md

```diff
@@ -0,0 +1,442 @@
+# Network Observability On Demand
+
+![logo](./images/cli_logo.png)
+
+By: Julien Pinsonneau
```

Comment:
@jotak & @skrthomas let's add you there ! Thanks for your help, correcting my franglsh and improving this article

```suggestion
By: Julien Pinsonneau 
Thanks to: Joël Takvorian and Sara Thomas for reviewing
```